### PR TITLE
Update Federation API ref docs for 1.11.

### PR DESCRIPTION
**Pull request by steveperry-53**
Previews

* [v1 Model Definitions](https://deploy-preview-9064--kubernetes-io-vnext-staging.netlify.com/docs/reference/federation/v1/definitions/)
* [v1 Operations](https://deploy-preview-9064--kubernetes-io-vnext-staging.netlify.com/docs/reference/federation/v1/operations/)
* [extensions/v1beta1 Model Definitions](https://deploy-preview-9064--kubernetes-io-vnext-staging.netlify.com/docs/reference/federation/extensions/v1beta1/definitions/)
* [extensions/v1beta1 Operations](https://deploy-preview-9064--kubernetes-io-vnext-staging.netlify.com/docs/reference/federation/extensions/v1beta1/operations/)

### CODE REVIEW

File: content/en/docs/reference/federation/v1/definitions.html [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: v1 Model Definitions
+title: v1  Model Definitions
```
Comment (mistyhacks) [inline content/en/docs/reference/federation/v1/definitions.html]: Nit: whitespace

**Comment by steveperry-53** [comment]
@mistyhacks I fixed the extra white space in "v1 Model Definitions". The other points you make are reasonable, but I think those should not be goals for this point in time. Right now I think we just need to get the content updated for 1.11. Then at some point between 1.11 and 1.12, we could make the kind of improvements you are suggesting.

**Pull request merged**